apps/protocol-frontend/src/components/AttestationsTableShell.tsx

```diff
@@ -10,137 +8,139 @@ import {
-                          </Text>
-                        </Stack>
-                        <Button
-                          bgColor="brand.primary.50"
-                          color="brand.primary.600"
-                          transition="all 100ms ease-in-out"
-                          _hover={{ bgColor: 'brand.primary.100' }}
-                          flexBasis="10%"
-                          colorScheme="brand.primary"
-                          disabled={selectedContributions?.length === 0}
-                          onClick={attestationsModalHandler}
-                          data-testId="attest-testId"
-                        >
-                          Attest
-                        </Button>
-                      </Stack>
-                    </Box>
-                    <AttestationsTable
-                      contributionsData={daoContributions}
-                      setSelectedContributions={setSelectedContributions}
-                    />
-                  </Stack>
-                </Box>
-              </TabPanel>
-              <TabPanel paddingX="0">
-                <Box
-                  background="white"
-                  boxShadow="sm"
-                  borderRadius={{ base: 'none', md: 'lg' }}
-                >
-                  <Stack spacing="5">
-                    <Box paddingX={{ base: '4', md: '6' }} paddingTop={4}>
-                      <Stack direction="column" gap="2">
-                        <Text fontSize="lg" fontWeight="medium">
-                          My Attestations
-                        </Text>
-                        <Text fontSize="md" fontWeight="normal">
-                          These are Contributions that you have already Attested
-                          to.
-                        </Text>
-                      </Stack>
-                    </Box>
-                    <Box width="100%" maxWidth="100vw" overflowX="auto">
-                      {daoContributions.length > 0 ? (
-                        <MyAttestationsTable
-                          contributionsData={daoContributions}
-                        />
-                      ) : (
-                        <EmptyContributions />
-                      )}
-                    </Box>
-                  </Stack>
-                </Box>
-              </TabPanel>
-            </TabPanels>
-          </Tabs>
-        ) : (
-          <EmptyContributions />
-        )}
-      </Box>
-      <ModalWrapper
-        name="bulkAttestationModal"
-        title="Attest to DAO Contributions"
-        localOverlay={localOverlay}
-        size="3xl"
-        content={<BulkAttestationModal contributions={selectedContributions} />}
-      />
-    </>
+    <Box
+      paddingY={{ base: '4', md: '8' }}
+      paddingX={{ base: '0', md: '8' }}
+      color="gray.700"
+      maxWidth="1200px"
+    >
+      <PageHeading>Attestations</PageHeading>
+      {isFetching && contributions && contributions.pages.length === 0 ? (
```

Comment:
Do we need this logic to be included inside the component due to the Infinite Scroll? In other components we split the loading/fetching states into their own return to streamline the logic -- something such as 

```
  if (isFetching || isLoading) {
    return <GovrnSpinner />;
  }
```

Wondering if maybe it's better to have inside the component like you did due to how Infinite Scrolls work

Reply:
I think splitting out is definitely cleaner. I have it in because that was how it was, but I will split out